lib/web/mage/gallery/gallery.less

```diff
@@ -762,9 +762,10 @@
     img {
         left: 0;
         max-width: inherit;
+        max-height: inherit;
         position: absolute;
         top: 0;
-        object-fit: scale-down;
+        object-fit: cover;
```

Comment:
Hi @krzksz 

The fix is not working properly.
The position of zoom image is not showing right area - see https://imgur.com/q4c25NP

Thanks

Reply:
Thanks for the QA, I'll look into it ASAP.

Reply:
The change you outlined was not needed, I tested magnification with different sample data images and now it works fine.